sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -209,33 +238,27 @@ class USReceipt(object):  # pylint: disable=too-many-instance-attributes
     :ivar fields:
         A dictionary of the fields found on the receipt.
     :vartype fields: dict[str, ~azure.ai.formrecognizer.FormField]
-    :ivar ~azure.ai.formrecognizer.PageRange page_range:
+    :ivar ~azure.ai.formrecognizer.FormPageRange page_range:
         The first and last page of the input receipt.
     :ivar list[~azure.ai.formrecognizer.FormPage] pages:
         Contains page metadata such as page width, length, text angle, unit.
         If `include_text_content=True` is passed, contains a list
         of extracted text lines for each page in the input document.
     :ivar str form_type: The type of form.
-    :ivar str receipt_locale: Defaults to "en-US".
     """
 
     def __init__(self, **kwargs):
+        super(USReceipt, self).__init__(**kwargs)
         self.merchant_address = kwargs.get("merchant_address", None)
         self.merchant_name = kwargs.get("merchant_name", None)
         self.merchant_phone_number = kwargs.get("merchant_phone_number", None)
-        self.receipt_type = kwargs.get("receipt_type", None)
         self.receipt_items = kwargs.get("receipt_items", None)
         self.subtotal = kwargs.get("subtotal", None)
         self.tax = kwargs.get("tax", None)
         self.tip = kwargs.get("tip", None)
         self.total = kwargs.get("total", None)
         self.transaction_date = kwargs.get("transaction_date", None)
         self.transaction_time = kwargs.get("transaction_time", None)
-        self.fields = kwargs.get("fields", None)
-        self.page_range = kwargs.get("page_range", None)
-        self.pages = kwargs.get("pages", None)
-        self.form_type = kwargs.get("form_type", None)
-        self.receipt_locale = kwargs.get("receipt_locale", "en-US")
```

Comment:
You should also delete receipt_type and receipt_locale from the repr below (this would also require the USReceipt repr test)

Reply:
nope - USReceipt inherits from RecognizedReceipt - so it will still stay logically in the repr

Reply:
you're right, my mind totally blanked there. thanks!